06-reactjs/components/movie-form.js

```diff
@@ -0,0 +1,32 @@
+var React = require('react');
+
+var MovieForm = React.createClass({
+    /*That clears the form fields when the form is submitted with valid input.*/
+    handleSubmit: function (event) {
+        event.preventDefault();
+        var title = React.findDOMNode(this.refs.title).value.trim();
+        var genre = React.findDOMNode(this.refs.genre).value.trim();
+        var rating = React.findDOMNode(this.refs.rating).value.trim();
+
+        if (!title || !genre || !rating) {
+            return;
+        }
+        this.props.onMovieSubmit({title: title, genre: genre, rating: rating});
+        React.findDOMNode(this.refs.title).value = '';
+        React.findDOMNode(this.refs.genre).value = '';
+        React.findDOMNode(this.refs.rating).value = '';
+        return;
+    },
+    render: function () {
+        return (
+            <form className="movieForm" onSubmit={this.handleSubmit}>
```

Comment:
When we have more than 1 property provided on the jsx we usually use a helper method to return the properties. Maybe you can refactor this to be getFormProps